### ISSUE TITLE
Feature request: Can recording objects have explicit experiment and recording number attributes?

### ISSUE THREAD

**Issue by holmescharles** [open-ephys/open-ephys-python-tools#9]
Recordings have an experiment_index and recording_index, but these are just enumerations determined at run time. I would like to have access to the numbers in the path.

E.g., 

If I have three recordings:
1. Experiment 1, Recording 1
2. Experiment 2, Recording 1
3. Experiment 2, Recording 2

I would like attributes that give me 1, 2, and 2 for experiments (not 0, 1, 1) and 1, 1, 1 for recording (not 1, 2, 3).

Thanks!

**Comment by jsiegle** [comment]
Agreed, this is an important thing to add. Not sure about the timeline for implementing this, but it will likely coincide with our next major GUI release (0.6.0) in February or March.
